test/controllers/time_entries_controller_test.rb

```diff
@@ -71,6 +75,16 @@ class TimeEntriesControllerTest < ActionController::TestCase
     assert_equal original_note, @time_entry.note
   end
 
+  test "redirects to today's entries after updating an entry for today" do
+    patch :update, id: @time_entry, time_entry: { start_time: 1.minute.ago.american_date }
```

Comment:
This is a really tiny nitpick, but this test will fail if run at midnight. Why not `{ start_time: Time.now.american_date }`?